common/workunit/workunit.cpp

```diff
@@ -5412,8 +5413,7 @@ void CWorkUnitFactory::importWorkUnit(const char *zapReportFileName, const char
                 throw MakeStringException(WUERR_InvalidUserInput, "Failed to find %s in %s.", mask, unzipDir.str());
             }
 
-            fileName.set(unzipDir).append(PATHSEPSTR);
-            di->getName(fileName);
+            fileName.set(di->query().queryFilename());
```

Comment:
NB: this is an incidental change, but cleaner and relates to removal of a spurious extra PATHSEPCHAR in the unzipDir (removed in another change).